kasmvnc/README.md

```diff
@@ -29,11 +31,24 @@ Automatically install [KasmVNC](https://kasmweb.com/kasmvnc) in a workspace, and
    module "kasmvnc" {
      source              = "https://registry.coder.com/modules/kasmvnc"
      agent_id            = coder_agent.example.id
-     version             = "1.0.0"
+     custom_version      = "1.0.0"
      desktop_environment = "mate"
      port                = 6080
    }
+   ```
+
+3. Use custom timezone and locale:
 
+   ```hcl
+   module "kasmvnc" {
+     source              = "https://registry.coder.com/modules/kasmvnc"
+     agent_id            = coder_agent.example.id
+     custom_version      = "1.0.0"
+     desktop_environment = "mate"
+     port                = 6080
+     locale              = "en_US.UTF-8"
+     timezone            = "America/Los_Angeles"
+   }
    ```
 
-![Screenshot of KasmVNC]() //TODO
+   > **Note:** It is recommended to pre-install a desktop environment in your Dockerfile/VM image to speed up the startup time of your workspace.
```

Comment:
🙌🏼 

I know you are removing this for now, but this module will be top-tier once we get it working

Reply:
I am sad as I have spent 10+ hours on trying to make it universal :/